worker/include/RTC/Transport.h

```diff
@@ -50,8 +50,7 @@ namespace RTC
 		void AddRtpReceiver(RTC::RtpReceiver* rtpReceiver);
 		void RemoveRtpReceiver(RTC::RtpReceiver* rtpReceiver);
 		void SendRtpPacket(RTC::RtpPacket* packet);
-		void SendRtcpPacket(RTC::RTCP::Packet* packet);
-		void SendRtcpPacket(RTC::RTCP::CompoundPacket* packet);
+		void SendRtcp(const uint8_t* data, size_t len);
```

Comment:
I don't like this signature. `Transport.SendRtpPacket()` receives a `RtpPacket` and uses its `GetRaw()` and `GetLength()`. I would like the same for RTCP `CompoundPacket` and `Packet` (for consistency).